NOTICE

```diff
@@ -1,46 +1,619 @@
+TOSCA" which was originally developed by Marcus Eisele as a bachelor thesis at the
+Institute of Architecture of Application Systems (IAAS) of the University of Stuttgart from
+April 2013 to October 2013. For more information see http://www.iaas.uni-stuttgart.de
+
+This product includes software from the project "OpenTOSCA Build Plan Generator"
+which was originally developed by Kálmán Képes as a bachelor thesis at the Institute of
+Architecture of Application Systems (IAAS) of the University of Stuttgart from January
+2013 to July 2013. For more information see http://www.iaas.uni-stuttgart.de
+
+This product includes software from the project "OpenTOSCA Service Invoker" which
+was originally developed by Michael Zimmermann as a bachelor thesis at the Institute of
+Architecture of Application Systems (IAAS) of the University of Stuttgart from January
+2013 to July 2013. For more information see http://www.iaas.uni-stuttgart.de
+
+This product includes software from the project "OpenTOSCA File Service" which was
+originally developed by Rene Trefft as a Bachelor Thesis at the Institute of Architecture of
+Application Systems (IAAS) of the University of Stuttgart from December 2012 to June
+2013. For more information see http://www.iaas.uni-stuttgart.de
+
+This product includes software from the project "Plan Invocation Framework" which was
+originally developed by Christian Endres as a bachelor thesis at the Institute of Architecture
+of Application Systems (IAAS) of the University of Stuttgart from November 2012 to May
+2013. For more information see http://www.iaas.uni-stuttgart.de
+
+This product includes software from the project "OpenTOSCA", whose first
+implementation was developed by Christian Endres, Matthias Fetzer, Markus
+Fischer, Nedim Karaoğuz, Kálmán Képes, Rene Trefft, and Michael Zimmermann
+as a study project ("Studienprojekt") at the Institute of Architecture of
+Application Systems (IAAS) and Institute for Parallel and Distributed
+Systems - Applications of Parallel and Distributed Systems (IPVS/AS) of
+the University of Stuttgart from 2011 to 2012.
+For more information see http://www.iaas.uni-stuttgart.de
+and http://www.ipvs.uni-stuttgart.de/as.html
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the
+terms of  the Apache Software License 2.0 which is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+== Source Code
+
+* https://github.com/OpenTOSCA/container
+
+== Copyright Holders
+
+Copyright (c) 2012-2017 University of Stuttgart
+Copyright (c) 2014-2015 Tobias Binz
+Copyright (c) 2012-2017 Uwe Breitenbücher
+Copyright (c) 2012-2017 Florian Haupt // TODO Check
+Copyright (c) 2012-2017 Oliver Kopp
+Copyright (c) 2012-2017 Alexander Nowak // TODO Check
+Copyright (c) 2012-2017 Sebastian Wagner // TODO Check
+Copyright (c) 2012-2017 Tim Waizenegger // TODO Check
+Copyright (c) 2012-2017 Marcus Eisele  // TODO Check
```

Comment:
Member of first StuPro 2012-2013 + Bachelor thesis (instance data API) until end of 2013.